cherryontech/jupiter-jumpers
# Add functioning heat map

## Description

1. Adds the heat map to the trends page, and updates the y axis custom ticks to be the logos instead of the colored dots. 

## Related Issue

N/A

## Related Story Card

[SCRUM-40](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-40?atlOrigin=eyJpIjoiNmUwMmYwZmY2OTRlNGY4MGFmY2RmMjc4YzcwYmE4NzMiLCJwIjoiaiJ9)

## Type of Changes

Feature and style update

## Acceptance Criteria

- [X] Given I have a substantial history of logs in the "Monthly View"
- [X] When I access the "Trends" section
- [X] Then the app displays:
- [X] My top six most frequent words used in my logs

## Update Screenshots

![Screenshot 2024-06-04 at 1 01 05 PM](https://github.com/cherryontech/Uncloud/assets/47403135/305ff3c6-a9e9-4dbe-8958-d32c742bb9f4)

## Testing Instructions

1. Add a bunch of reflections with repeating words (so that the heat map demonstrated functionality). E.g., "love love love love" on several moods. 
2. Ensure that the heat map reflects the frequency of the words
3. Try resizing the window, and ensure the charts resize correctly.
